src/bin/utils/console.ts

```diff
@@ -12,7 +12,7 @@ const rl = createInterface({
 rl.on('line', line => {
   const log = JSON.parse(line)
 
-  if (log.msg === 'status') {
+  if (log.msg === 'console') {
```

Comment:
Typically, rename commits should not make code changes at the same time. No need to fix, just a note for the future.

Reply:
Bugger.. I thought I was careful about refactoring this out. Doubly noted.